src/extension.ts

```diff
@@ -75,6 +86,41 @@ export function activate(context: vscode.ExtensionContext) {
 	let dartIndentFixer = new DartIndentFixer();
 	context.subscriptions.push(vscode.window.onDidChangeActiveTextEditor(td => dartIndentFixer.onDidChangeActiveTextEditor(td)));
 	dartIndentFixer.onDidChangeActiveTextEditor(vscode.window.activeTextEditor); // Handle already-open file.
+
+	context.subscriptions.push(vscode.workspace.onDidChangeConfiguration(handleConfigurationChange));
+}
+
+function handleServerStatus(status: ServerStatusNotification) {
```

Comment:
I think it'd be worth pulling all this into another file/class to keep `extension.ts` clean as it's a good place to understand how everything it's wired up but this will clutter it a little.